code401challenges/src/main/java/code401challenges/mergeSort/MergeSort.java

```diff
@@ -0,0 +1,56 @@
+package code401challenges.mergeSort;
+
+public class MergeSort {
+    public static void sort(int[] arrIn, int left, int right){
```

Comment:
the first piece is that your sort should be able to be called directly on an array without passing in a left and right

Reply:
to make this all easier lets return the sorted array instead of void